msg/templates/urtps/microRTPS_transport.cpp

```diff
@@ -223,18 +224,13 @@ ssize_t Transport_node::write(const uint8_t topic_ID, char buffer[], size_t leng
 	header.crc_h = (crc >> 8) & 0xff;
 	header.crc_l = crc & 0xff;
 
-	ssize_t len = node_write(&header, sizeof(header));
-
-	if (len != sizeof(header)) {
-		goto err;
-	}
-
-	len = node_write(buffer, length);
-
-	if (len != ssize_t(length)) {
+	char buffer_sync[MAX_BUFFER_SIZE];
```

Comment:
This changes stack size requirements

Reply:
That's why dynamic allocation might be ideal way to do, What do you suggest ?

Reply:
Might be, yes. But we only need it for the UDP case (if we go for my first proposed solution). And the allocation only needs to be done on startup (or to increase the buffer size when required).